src/ripple/protocol/Rate.h

```diff
@@ -43,7 +44,13 @@ struct Rate
     Rate (std::uint32_t rate)
         : value (rate)
     {
```

Comment:
I feel like this constructor really ought to be `explicit`.  That change may have too many consequences for this pull request, but we should consider it for the future.